demo/src/MemoryModelsMenu.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { ReactNode, useState } from "react";
+import { Box, Button, Menu } from "@mui/material";
+import ExpandMoreRoundedIcon from "@mui/icons-material/ExpandMoreRounded";
+
+type MemoryModelsMenuPropTypes = {
+    menuName: string;
+    testId: string;
+    menuItems: ReactNode;
+};
+
+export default function MemoryModelsMenu(props: MemoryModelsMenuPropTypes) {
+    const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+    const open = Boolean(anchorEl);
+    const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+        setAnchorEl(event.currentTarget);
+    };
+    const handleClose = () => {
+        setAnchorEl(null);
+    };
+
+    return (
+        <>
+            <Button
+                onClick={handleClick}
+                data-testid={props.testId}
+                sx={{
+                    textTransform: "none",
+                    "& .MuiSvgIcon-root": {
```

Comment:
Adding this CSS is fine, but do so in the `styles.css` file. You might need to define a custom class to ensure the rule only applies to "menu open" buttons.